backend/src/main/java/net/gunn/elimination/model/Kill.java

```diff
@@ -0,0 +1,49 @@
+package net.gunn.elimination.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Map;
+
+@Entity
+public class Kill implements Serializable {
+	@Id
+	@GeneratedValue(strategy = GenerationType.AUTO)
+	@JsonIgnore
+	private String id;
+
+	@JsonProperty
+	private String eliminator;
+
+	@JsonProperty
+	private String eliminated;
+
+	@JsonProperty
+	private String timeStamp;
```

Comment:
maybe use java.sql.Date instead, since it'll work nicely with potential future SQL queries